pkg/consts/monitoring.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: extension-cri-resmgr-monitoring-scrape-config
+  namespace: {{ namespace }}
+  labels:
+    extensions.gardener.cloud/configuration: monitoring
+data:
+  scrape_config: |
+    - job_name: cri-rm
+      honor_timestamps: true
+      scrape_interval: 1m
```

Comment:
it should be in sync, with --metrics-interval argument passed to cri-resource-manager - it is worth to add a comment about it here.

Reply:
I added the comment